eslint.config.js

```diff
@@ -0,0 +1,32 @@
+'use strict'
+
+const neo = require('neostandard')
+
+module.exports = [
+  {
+    ignores: [
```

Comment:
You don't need these separate ignores if you provide them to neostandard as well, the neostandard ones are set as global ignores 🙂

Reply:
done :)